data/test.txt.dvc

```diff
@@ -0,0 +1,5 @@
+outs:
+- md5: a1cab28ad8554fd9269d31e5708469d0
+  size: 32724891
+  hash: md5
+  path: test.txt
```

Comment:
Is there a reason this is here but not in the dvc yaml pipeline?

Reply:
Yes there is, this way when pulling the dvc it should pull the test.txt, train.txt and val.txt from the google drive and put them in the data directory

Reply:
Hmm but it is explicity said in the assignment to have it in the pipeline: The dataset is stored outside the project and can be automatically downloaded as part of the pipeline.
Is there a way to do that instead?

Reply:
I have added it to the pipeline, however after this discussion I am not sure if getting a local copy of the remote dataset (even when everything is automatically done) is the solution that we want. I think we should look into the possibility of not getting the dataset remotely. However I think that should be a different issue for now.

Reply:
I think you should be downloading the dataset from kaggle like you did at first maybe in another file and then just add that to the pipeline, this will save the data in the data folder and remotely. Then the get_data.py file can have those files as dependencies and that file transforms it to the raw data.

Reply:
I've also heard from another group that they are downloading just from their google drive...that could be an option.

Reply:
We are downloading it from the google drive right now that is what specifically the something.txt.dvc are for. So if you do the dvc pull you will get download them from the drive. They are then downloaded into the respective something.txt files. Which the pipeline now uses, so it is stored locally in the data folder and remotely in the google drive. I am talking specifically about not storing them locally (if that is possible) and **only** storing them remotely.

Reply:
But then the downloading isnt a part of the pipeline...its a part of dvc pull. Which also has had a lot of bugs

Reply:
Hmm yeah that is fair. I do not know how to do that right now. Because when adding the txt files to the drive they automatically added this and the .gitignore

Reply:
I think you should just add the downloading from kaggle part again

Reply:
I disagree with that, I think that this is better even though we need a dvc pull. But we could make a new issue for that as if you want. Because I think we also want to keep the datasets added to the drive.

Reply:
I think we should merge this as is and create a new issue for that, as how I understand it now kaggle does not interact well with the dvc pipeline.

Reply:
Okay sure but we could also just download it from the drive, we don't need kaggle